### PR TITLE
K8s Poseidon release notes

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: DOC-2242 DOC-2363 DOC-2370 DOC-2371
Preview: 

- https://docs.redis.com/staging/DOC-2242/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd/
- https://docs.redis.com/staging/DOC-2242/kubernetes/reference/supported_k8s_distributions/

### CODE REVIEW

**Review by cmilesb — APPROVED** [review]

File: content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+* Improved log collector execution log, filtering out empty files, capturing volume attachments, option to disable RS capture debug info package
+* Added `extraEnvVars` field in REC to support environment variables for RS pods
+
+## Fixed bugs
+
+* REC in invalid state can't be deleted (RED-78124)
+* OpenShift route not deleted after REC is deleted (RED-94121)
+* REAADB secret status not being updated in source cluster (RED-96296)
+* Invalid REAADB that is not rejected by admission might be get deleted after apply (RED-96300)
+* Opeartor logs an error about missing permissions to nodes on startup (RED-98227)
```
Comment (rrelledge) [inline content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md:63]:
```suggestion
* Operator logs an error about missing permissions to nodes on startup (RED-98227)
```

**Review by rrelledge — DISMISSED** [review]
Some small typos

File: content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+categories: ["Platforms"]
+aliases: [ 
+    /kubernetes/release-notes/6-4-2-releases/k8s-6-4-2-tbd/,
+      ]
+---
+## Overview
+
+The Redis Enterprise K8s 6.4.2-tbd release supports Redis Enterprise Software 6.4.2 and contains new features and feature improvements.
+
+The key features, bug fixes, and known limitations are described below.s
```
Comment (rrelledge) [inline content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md:16]:
```suggestion
The key features, bug fixes, and known limitations are described below.
```

File: content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+* Added support for K8s 1.26 for AKS and GKE
+* Improved log collector execution log, filtering out empty files, capturing volume attachments, option to disable RS capture debug info package
+* Added `extraEnvVars` field in REC to support environment variables for RS pods
+
+## Fixed bugs
+
+* REC in invalid state can't be deleted (RED-78124)
+* OpenShift route not deleted after REC is deleted (RED-94121)
+* REAADB secret status not being updated in source cluster (RED-96296)
+* Invalid REAADB that is not rejected by admission might be get deleted after apply (RED-96300)
```
Comment (rrelledge) [inline content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md:62]: This sentence was a bit confusing. Aside from the verb fix, should "admission" be "admission controller" and should "after apply" be either "after application" or "after applying" something specific? 
```suggestion
* Invalid REAADB that is not rejected by admission might be deleted after apply (RED-96300)
```

File: content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+* Log collector fails to include RS debuginfo when old kubectl is used (RED-101170)
+* REC creations fails due to required LDAP configuration in OLM form view (RED-100517)
+* Upgrading to 6.4.2 (or 6.2.18) on OpenShift will fail if pod 0 is not the master node when upgrade starts (RED-102100)
+
+## API changes
+
+The following fields were changed in the [RedisEnterpriseCluster (REC) API](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/redis_enterprise_cluster_api.md):
+  * `containerTimezoneSpec.timezoneName` added to configure timezones on the Redis Enterprise pods
+  * `hostAliases` added for adding an entry to the Redis Enterprise pods' `/etc/hosts`
+  * `redisEnteprisePodAnnotations` added to specify annotations that should only be4 set on the Redis Enterprise pods
```
Comment (rrelledge) [inline content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md:75]:
```suggestion
  * `redisEnteprisePodAnnotations` added to specify annotations that should only be set on the Redis Enterprise pods
```

File: content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+\* Support added in this release
+
+## Upgrade to 6.4.2-tbd
+
+Be aware the following changes included in this release affect the upgrade process. Please read carefully before upgrading to 6.4.2-tbd.
+
+* **Supported upgrade paths**
+
+  If you are using a versions earlier than 6.2.10-45, you cannot upgrade directly to this release. You must upgrade to 6.2.10-45 before you can upgrade to versions 6.2.18 or later.
```
Comment (rrelledge) [inline content/kubernetes/release-notes/6-4-2-releases/6-4-2-tbd.md:117]:
```suggestion
  If you are using a version earlier than 6.2.10-45, you cannot upgrade directly to this release. You must upgrade to 6.2.10-45 before you can upgrade to versions 6.2.18 or later.
```

File: content/kubernetes/active-active/create-aa-crdb-cli.md [hunk 6]
```diff
@@ -165,11 +165,14 @@ For each cluster, verify the VirtualService resource has two `- match:` blocks i
...
 
     If the rec name was modified, reapply [scc.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/openshift/scc.yaml) to the namespace to reestablish security privileges.
+    
     ```sh
     oc apply -f scc.yaml
-    oc adm policy add-scc-to-group redis-enterprise-scc  system:serviceaccounts:<namespace>
+    oc adm policy add-scc-to-group redis-enterprise-scc-v2  system:serviceaccounts:<namespace>
     ```
 
+    Releases before 6.4.2-6 use the earlier version of the SCC, named `redis-enterprise-scc-v2`.
```
Comment (randv1r) [inline content/kubernetes/active-active/create-aa-crdb-cli.md:174]: Here it should be without the "v2", i.e. "redis-enterprise-scc".

**Pull request merged**